templates/ngx-service.mustache

```diff
@@ -17,20 +17,25 @@ interface HttpOptions {
   withCredentials?: boolean,
 }
 
+declare var window;
```

Comment:
There is no need to declare `window` here, as the code will be generated for the browser.